src/App.js

```diff
@@ -112,6 +113,7 @@ const App = ({ locale, location, dependencies: { appSessionRef, sessionManager }
             exact
             component={ReportsPartialsCampaigns}
           />
+          <PrivateRoute path="/plan-calculator" exact component={PlanCalculator} />
```

Comment:
Is _private_ ok? I mean, only authenticated users can open it?

Reply:
Yes, you have to be logged to make an upgrade or buy credits.